diff --git a/telephony/java/com/android/internal/telephony/PhoneFactory.java b/telephony/java/com/android/internal/telephony/PhoneFactory.java
index 7f1ab781f0e..7d653a0a13e 100644
--- a/telephony/java/com/android/internal/telephony/PhoneFactory.java
+++ b/telephony/java/com/android/internal/telephony/PhoneFactory.java
@@ -137,6 +137,9 @@ public class PhoneFactory {
                 } else if ("motow3g".equals(sRILClassname)) {
                     Log.i(LOG_TAG, "Using Motorola Wrigley 3G RIL");
                     sCommandsInterface = new MotoWrigley3GRIL(context, networkMode, cdmaSubscription);
+                } else if ("HuaweiQualcommRIL".equals(sRILClassname)) {
+                    Log.i(LOG_TAG, "Using Huawei Qualcomm RIL");
+                    sCommandsInterface = new HuaweiQualcommRIL(context, networkMode, cdmaSubscription);
                 } else {
                     sCommandsInterface = new RIL(context, networkMode, cdmaSubscription);
                 }
